cdap-ui/app/directives/widget-container/widget-multi-keyvalue/widget-multi-keyvalue.js

```diff
@@ -104,6 +108,7 @@ angular.module(PKG.name + '.commons')
         init();
         addPropertiesFromModel();
 
```

Comment:
let key = (property.enableCustomKeyInput) ? property.keyInputValue : property.keySelectValue.id;
please check null or empty before that

Reply:
these values will never be `null` and in case of empty it won't be a problem.